test/php/model/languageforge/lexicon/command/LexProjectCommandsTest.php

```diff
@@ -219,4 +219,63 @@ public function testCreateCustomFieldsViews_CreateTwoCustomFieldViewsViaRunClass
         $customField1 = $project2->config->roleViews[LexRoles::MANAGER]->fields[$customFieldName];
         $this->assertTrue(is_a($customField1, 'Api\Model\Languageforge\Lexicon\Config\LexViewFieldConfig'));
     }
+
+    private function runCustomFieldRenameTest($newName) {
+        $environ = new LexiconMongoTestEnvironment();
+        $environ->clean();
+
+        // setup: 1 example custom field (existing), 1 in senses (to delete), 1 in entry (to create)
+        $project = $environ->createProject(SF_TESTPROJECT, SF_TESTPROJECTCODE);
+        $customFieldNameExisting = 'customField_examples_testOptionList';
+        $viewFieldConfig = new LexViewFieldConfig();
+        $viewFieldConfig->type = 'MultiUnicode';
+        $viewFieldConfig->show = false;
+        $project->config->roleViews[LexRoles::MANAGER]->fields[$customFieldNameExisting] = $viewFieldConfig;
+        $customFieldNameToCreate = 'customField_entry_testMultiText';
+        $customFieldSpecs = [
+            [
+                'fieldName' => $customFieldNameExisting,
+                'fieldType' => 'MultiUnicode'
+            ]
+        ];
+        $mangerRoleViewFieldCount = $project->config->roleViews[LexRoles::MANAGER]->fields->count();
```

Comment:
```suggestion
        $managerRoleViewFieldCount = $project->config->roleViews[LexRoles::MANAGER]->fields->count();
```
fix spelling to "manager"